### ISSUE TITLE
can't do multiple searches on the same file

### ISSUE THREAD

**Issue by yuipcheng** [everettjf/vscode-filter-line#6]
after the first search for a file is done, then i tried to search for a different string, but after pressing enter, the old result is still there. i have to exit vs code, old result file is gone, and then come back to do another search.

**Comment by everettjf** [comment]
Yes, it may lose focus on the generated file after each filtering.

e.g.
When filtering file `foo.log`, the 1st filter action generates `foo.log.filterling.log`.
NOW, if we continue filter on the  `foo.log.filterling.log`, the 2nd result is empty, which is a bug.

BUT, we could fix(skip) the bug by `Click` the result `foo.log.filterling.log`.
SO, the order is: 
1. 1st filter
2. click the result file in the vscode
3. 2nd filter

I will try to fix it in a few days.

**Comment by lovettchris** [comment]
I'm hitting this too, I'd love to be able to change the filter rather than generating a new filterline.log file for every filter.  For example I have log files with interleaved output for each thread, and I want to see thread 1, then 2, then 3 and so on quickly re-filtering for each.  It's nice to have a separate view so I can also see the original file side by side.  But having the filter operation only work once is a show stopper for me.

**Comment by everettjf** [comment]
@lovettchris Thanks for your response. I'm on it.
Is "thread 1, then 2, then 3 and so on" means something like "filterline.log, filterline.1.log, filterline.3.log and so on" ?

**Comment by everettjf** [comment]
Which one do you prefer?
1. "filterline.log, filterline.1.log, filterline.3.log and so on"
2. repeated filtering on the same file "filterline.log"

**Comment by lovettchris** [comment]
(2) repeated filtering on the same file, so I can open the original file, filter it, see the results, and repeat, without having to close a bunch of new windows every time...

**Comment by everettjf** [comment]
v1.2.2 published. please update and try.

**Comment by NARBEDA** [comment]
I doesn't work for me! I still must reopen VS Code for next filtering. Am I doing something wrong?

**Comment by everettjf** [comment]
HI , @NARBEDA 
1. reopen vscode ? or click the result buffer ?
2. Windows or macOS ?

**Comment by NARBEDA** [comment]
Yes, I've to reopen vscode, to do the next search. I only can do one search! It's on Windows! Are there some special Settings?

**Comment by chraac** [comment]
v1.2.2 still doesn't work! and here to reproduce:
1. filter with regex
2. close filter result
3. filter with another regex

And it's windows 7 with vscode 1.31.1

**Comment by everettjf** [comment]
Hi all, I have temporarily fixed the bug on Windows.  Please update and retry.
(But still exists a little problem, repeated filter action will produce some temporary files, when you close vscode, these files will auto-removed. I still do not know the reason and will work on it.

**Comment by lovettchris** [comment]
I have 1.2.8 and still only works once on a given file.

**Comment by chraac** [comment]
Thanks to reply, and bug still exist in the same routine with 1.2.8

**Comment by everettjf** [comment]
Hello everyone, now I can say "the bug is fixed" with 1.2.10
PS: 
```
        let writeStream = fs.createWriteStream(outputPath);
```
The `writeStream` need to be `closed` explicity on Windows.

**Comment by NARBEDA** [comment]
Works perfect! Many thanks!
